docs/reference/mapping/params/eager-global-ordinals.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[eager-global-ordinals]]
+=== `eager_global_ordinals`
+
+Global ordinals is a data-structure on top of doc values, that maintains an
+incremental numbering for each unique term in a lexicographic order. Each
+term has a unique number and the number of term 'A' is lower than the
+number of term 'B'. Global ordinals are only supported on
```

Comment:
I think "Global ordinals are only used with `keyword` and `text` fields. In `keyword` terms they are available by default but `text` fields can only use them when `fielddata`, with all of its associated baggage, is enabled."

Or something like that.